doc/developers/advanced_installation.rst

```diff
@@ -186,7 +186,11 @@ commands in ``cmd`` or an Anaconda Prompt (if you use Anaconda):
 .. prompt:: bash C:\>
 
     SET DISTUTILS_USE_SDK=1
-    "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64
+    "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64
```

Comment:
I am wondering if this is still needed now that we use meson ... the `DISTUTILS_USE_SDK` is not needed anymore with 99% probability, the `vcvarsall.bat` is probably not needed.

Personally, on Windows I compile inside a miniforge prompt and I never do this ...

It would be great if you could confirm @vitorpohlenz!

Reply:
I also tried to install CPython from python.org and use a venv in a cmd console and same thing it works fine without setting `DISTUTILS_USE_SDK` and `vcvarsall.bat`. I opened https://github.com/scikit-learn/scikit-learn/pull/31202 to simplify the Windows instructions.

Reply:
Thanks @vitorpohlenz for confirming this!